crux-mir/README.md

```diff
@@ -135,14 +135,23 @@ To compile and test a single Rust program:
 * Clone this directory with `git clone https://github.com/GaloisInc/crucible.git` as we will need the `report-coverage` utility tool
 * In your crate run:
   ```
-  $ cargo crux-test --lib --  --branch-coverage --path-sat --output-directory test-coverage
+  $ EXPORT_ALL=true cargo crux-test --lib --  --branch-coverage --path-sat --output-directory test-coverage
```

Comment:
I'm not super keen on making `EXPORT_ALL` a part of the user requirements for doing coverage reporting, as this environment variable is mostly intended to be an [internal implementation detail of `mir-json`](https://github.com/GaloisInc/mir-json/blob/dc0fb66d47d2eb0e320efb25dcedcbba510cdd2e/doc/rustc.md?plain=1#L84-L96). I wonder if we should instead add another flag to `crux-test` that turns on `EXPORT_ALL=true` and generates a `crux-mir` wrapper script that enables `--branch-coverage`.